### ISSUE TITLE
Request sample structure to have the login page connected to the dashboard

### ISSUE THREAD

**Issue by ggunders** [akveo/blur-admin#110]
There are a number of issues with people trying to integrate the login with the SPA dashboard. perhaps some simple modifications would show how to integrate a login with the dashboard. Feature request for implementation or documentation suggestions. thanks

**Comment by vazh** [comment]
mine works fine.

**Comment by vazh** [comment]
you need to make nested routing, and ui-router does support that.
- login
- app
  - dashboard
  - etc.

**Comment by ggunders** [comment]
ok i got you, did exactly that.

now that i have a login module and the routing setup i want to have the auth.css styling applied to the login page. when you remove the auth.html you loose the auth.css. i can manually link the auth.css in the index.html and it looks fine. 

in a seperate issue it was said to wrap the login html code in a .loginwrapper DIV. do you also wrap all the auth.scss stuff in a wrapper as well to pull it in automatically. i tried that but that didnt help at all. im clearly missing something with sass dev.

basically how do i get the blur theme to work with the login page(notice im not using the included auth.html) thx

**Comment by vazh** [comment]
yes, both html and scss have to be wrapped in a class .loginwrapper.

im not using auth.html too, just the bit part between body tag.

the thing is, you need to rename auth.scss into the login page html filename, so it can connect. i dont understand, i just try it, and it works. dont forget to use _ too.

so, if your login page html filename is `login.html` rename your auth.scss into `_login.scss`

**Comment by 94rajesh** [comment]
@vazh given step by step procedure to create login with controller....thanks

**Comment by vazh** [comment]
@94rajesh sorry, but not right now.
i'll try to fork and update it later, sorry

**Comment by ggunders** [comment]
i named my login folder auth/auth.html and auth/AuthCtrl.js etc.(removing auth.html in the top level)

then changed auth.scss to _auth.scss

added the wrapper around the entire login html code in auth.html
added the wrapper around the entire _auth.scss code just after the imports

the styling is off. all gray. do i have to move the _auth.scss to a specific folder?

**Comment by ggunders** [comment]
disregard last, i got it. forgot to enable the blur theme in the theme-config.

thx guys. login stuff works

**Comment by vazh** [comment]
@ggunders help them by closing this issue.

**Comment by 94rajesh** [comment]
ggunder and vazh. how to change the page top after sign in new page

**Comment by SJ18** [comment]
hi @ggunders can u please share the source of your working login

**Comment by chanakaDe** [comment]
@vazh @ggunders  Can anyone please share a repo of the working login demo :+1:  I'm trying too and no good so far. It's better for everyone if someone can make a working login demo and share the link :+1:

**Comment by patrickgodinho** [comment]
@ggunders can you show your code?

**Comment by resko** [comment]
@ggunders I am interested too, can you provide a sample please ?
